levels/sre.tex

```diff
@@ -0,0 +1,368 @@
+
+    \bigbreak
+
+    Understands different networking layers (OSI model).
+
+    \bigbreak
+
+    Understands and can deploy overlay networks with routing and DNS resolution.
+    &
+
+    % Security
+    Understands operating systems security principles like iptables,
+    users/groups, file permissions, and capabilities. Can apply operating system
+    hardening guides like CIS Benchmarks.
+
+    \bigbreak
+
+    Understands basic security principles like SSH keys and TLS certificates.
+    &
+
+    % Systems Engineering
+    &
+
+    % SRE
+    Understands and can make changes to existing Makefiles, shell scripts, and
+    Dockerfiles.
+
+    \bigbreak
+
+    Understands declarative configuration and can use tooling like Terraform or
+    a Kubernetes operator to make changes to existing infrastructure and
+    applications.
+
+    \bigbreak
+
+    May have certification like Certified Kubernetes Administrator (CKA).
+    \\ [13em]
+\hline
+    % Level
+    2
+    &
+
+    % Product Output
+    \cellcolor{blue!25}
+    Can write high quality user and product focused documentation.
+    &
+
+    % Communication
+    Provides constructive review on peers' code and design. Helps new team
+    members during their first weeks.
+    &
+
+    % Networking
+    Can independently troubleshoot most network issues.
+
+    \bigbreak
+
+    Can setup and configure production quality network infrastructure like DNS,
+    Load Balancers, and encrypted networks (like IPsec or WireGuard). Can reason
+    about their reliability.
+    &
+
+    % Security
+    Understands and can apply basic cryptographic principles. Can configure SSH
+    and TLS for a server (chooses appropriate key sizes, algorithms, and versions),
+    pick strong authentication and authorization primitives, and appropriate
+    encryption for data in transit and rest.
+
+    \bigbreak
+
+    Can setup production quality encrypted networking (like IPsec or
+    WireGuard). Can reason about their reliability.
+    &
+
+    % Systems Engineering
+    &
+
+    % SRE
+    Excellent systems troubleshooter.
+
+    \bigbreak
+
+    Familiar with and can configure alerts for system performance (CPU, memory,
+    disk, networking utilization). Uses modern BPF tools for troubleshooting.
+
+    \bigbreak
+
+    Is a member of on-call rotation and can resolve issues outlined in
+    runbooks.
+
+    \bigbreak
+
+    Demonstrates knowledge of AWS. May have certification like AWS Certified
+    SysOps Administrator.
+    \\ [13em]
+\hline
+    % Level
+    3
+    &
+
+    % Product
+    \cellcolor{blue!25}
+    Collaborates with the team to scope requirements, based on good
+    understanding of existing longer term product vision and estimates of the
+    system design of a feature of a product.
+    &
+
+    % Communication
+    Supports less experienced peers' technical skills, answering questions and
+    being a resource. Documents and improves team practices.
```

Comment:
I'd move `Documents and improves team practices` to lvl 2. It is a relatively basic skill and does not always require particularly deep knowledge or experience, especially given that fact that more senior folks could provide feedback.